src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -3857,5 +3857,28 @@ static void M()
             var compilation = CreateCompilationWithMscorlib45(source, references: new[] { reference });
             compilation.VerifyEmitDiagnostics();
         }
+
+
+        [Fact]
+        [WorkItem(13734, "https://github.com/dotnet/roslyn/issues/13734")]
+        public void TypeofAwaitedTask()
+        {
+            string source =
+    @"
+using System.Threading.Tasks;
+using System;
+
+namespace AsyncBug
+{
+    public class Foo {
```

Comment:
Please avoid `Foo` in bug repros.  It's silly but it's a policheck violation that we have to excuse later on.  Easier to just avoid it.

Says the guy who had to run poli-check on all our source code before moving to GitHub ancd dealing with excusing the couple thousand references to `Foo`, `Bar` and `FooBar` that we had in the code base.